src/theme/mod.rs

```diff
@@ -67,6 +67,22 @@ impl Options {
             let exts = Box::new(NoFileStyle);
             return Theme { ui, exts };
         };
+
+        #[cfg(windows)]
+        if let Ok(()) = nu_ansi_term::enable_ansi_support() {
+            // succesfully enabled ansi support
+            // (ENABLE_VIRTUAL_TERMINAL_PROCESSING)
+        } else {
```

Comment:
Since only the else branch is doing something, shouldn't use:
```rust
if nu_ansi_term::enable_ansi_support().is_err() {
```
Otherwise this looks fine to me.